fs/sync/sync.go

```diff
@@ -937,6 +960,15 @@ func CopyDir(ctx context.Context, fdst, fsrc fs.Fs, copyEmptySrcDirs bool) error
 	return runSyncCopyMove(ctx, fdst, fsrc, fs.DeleteModeOff, false, false, copyEmptySrcDirs)
 }
 
+// CopyFiles copies files from  fsrc/remoteSrc into fdst/remoteDst.
+func CopyFiles(ctx context.Context, fdst fs.Fs, remoteDst string, fsrc fs.Fs, remoteSrc string, copyEmptySrcDirs bool) error {
```

Comment:
Maybe it would be more flexible / future proof to export `newSyncCopyMove` and `run` function?